ansible.cfg

```
@@ -0,0 +1,35 @@
+[defaults]
+inventory      = ./inventory/
+forks          = 5
+remote_user    = root
+roles_path     =./roles
+ask_pass       = True
+#log_path       =/var/log/ansible.log
+inventory_ignore_extensions = ~, .swp, .md, .adoc, .csv, .retry, .pyc, .pyo, .example
+display_skipped_hosts = False
+deprecation_warnings = False
+command_warnings = False
+nocows         = 1
+retry_files_enabled = False
+host_key_checking = False
+collections_paths = /home/rsawyer/git/rh
```

Would expect project specific relative path:

```suggestion
collections_paths = ./collections
```

---

This file has been deleted